src/Persistent/PersistentMockRegistry.php

```diff
@@ -2,17 +2,33 @@
 
 namespace Hammock\Persistent;
 
+use namespace HH\Lib\{C, Vec};
 use type Hammock\Interfaces\IDeactivatable;
 
 class PersistentMockRegistry {
+	const REGISTRY_SOFT_LIMIT = 100;
```

Comment:
Does this typecheck in `// strict` mode?
Shouldn't it require it to be `const int REGISTRY_SOFT_LIMIT = 100`?

Reply:
That's a good question.

I found some docs in the Hack Constants Docs that say
> If a class constant's type is omitted, it is inferred from the initializer, which must be present. In this case, that type is string.

https://docs.hhvm.com/hack/classes/constants

So I guess since the initializer is an int constant it infers that.

Explicitly adding types sounds nice though, I pushed a commit with some int types.